gstd/gstd_parser.c

```diff
@@ -342,7 +342,7 @@ gstd_parser_pipeline_create (GstdSession * session, gchar * action, gchar * args
 
   uri = g_strdup_printf ("/pipelines %s", args ? args : "");
 
-  ret = gstd_parser_parse_raw_cmd (session, "create", uri, response);
+  ret = gstd_parser_parse_raw_cmd (session, (gchar*)"create", uri, response);
```

Comment:
Can `gstd_parser_parse_raw_cmd` receive a `const gchar` instead?

Reply:
Yes, but if it is changed, also must be changed 'GstdClientCmd' array and also all the command functions (gstd_client_cmd_*) to keep coherent the code.